claat/render/md.go

```diff
@@ -243,24 +243,22 @@ func (mw *mdWriter) itemsList(n *types.ItemsListNode) {
 }
 
 func (mw *mdWriter) infobox(n *types.InfoboxNode) {
-	// TODO: This should use the "detail item" syntax so that it can be pure MD and not HTML
-	// kind
-	// : <content>
-	//
-	// The main issue is that when you do write(n.Content.Nodes...) it always adds two newlines
-	// at the beginning. This is because the gdoc parser parses info boxes as a list node
-	// followed by a text node, and the list node is an md block unto itself, resulting in two
-	// newlines being added to the output before the text.
+	// InfoBoxes are comprised of a ListNode with the contents of the InfoBox. 
+	// Writing the ListNode directly results in extra newlines in the md output
+	// which breaks the formatting. So instead, write the ListNode's children 
+	// directly and don't write the ListNode itself.
+	ln, ok := n.Content.Nodes[0].(*types.ListNode)
```

Comment:
I actually have two options here:

1) write the ListNode's children directly, and don't write the ListNode itself
2) modify the ListNode so it's no longer a block, and then write the ListNode itself

What's implemented here is (1). Is that better than (2)?

Reply:
I've thought about it a bit and I think (2) is more work than it's worth, and I can't find a good reason that it's better than (1), so I'm just gonna stick with (1).